external-testing/README.md

```diff
@@ -0,0 +1,60 @@
+# External Testing
+
+Let's talk about testing process against external resources. What does that
+mean? Well, talking to GitHub API, COPR API, running `fedpkg`, cloning
+repositories...
+
+
+## Use cases
+
+* Interact with an external API (such as GitHub, COPR, Pagure, FAS, k8s)
+  * Authenticated vs. unauthenticated calls.
+* Clone a git repository.
+* Construct a git repository
```

Comment:
What about push -- we can use it for creating PRs that are part of multiple jobs.

Reply:
I'd say push is covered by: run external command && R/W actions on a remote service